WalletWasabi/Tor/TorProcessManager.cs

```diff
@@ -233,7 +284,7 @@ internal virtual ProcessAsync StartProcess(string arguments)
 			Logger.LogDebug($"Environment variable 'LD_LIBRARY_PATH' set to: '{env["LD_LIBRARY_PATH"]}'.");
 		}
 
-		Logger.LogInfo("Starting Tor process ...");
+		Logger.LogInfo("Starting Tor process…");
```

Comment:
+1, also the default Windows `cmd` doesn't print it out, while the `Git Bash` and `PowerShell` does.

Reply:
> I have never seen this character before. Are you sure everyone will have this on their computer?

.NET has unicode strings (UTF-16) and unicode is de-facto standard these days.

Bash does not have issues with unicode (https://stackoverflow.com/a/602924), Bitcoin Core has a [merged PR](https://github.com/bitcoin/bitcoin/pull/21836/files) where they replaced three dots with ellipsis too.

`cmd` is old and that's why https://github.com/microsoft/terminal/ is being pushed as a default terminal on Windows to replace `cmd` which cannot be easily changed because it would break too many programs.

tl;dr `cmd` is an exception, unicode support in logs is not a problem.

Reply:
Anyway, I can revert it if you don't like it. Still I do not expect any issues come from this.